.gitignore

```
@@ -12,6 +12,7 @@ tmp/
 local.properties
 .loadpath
 .recommenders
+.DS_Store
```

```suggestion

################## Mac OS X ######################
.DS_Store
.DS_Store?
.AppleDouble
.LSOverride
```

reference: https://github.com/github/gitignore/blob/main/Global/macOS.gitignore